README.md

```diff
@@ -1,2 +1,15 @@
 # blog
 Blog posts from PH for PH and the rest of the world :)
+
+Formating of blog posts:
+
+When you first make a blog post make sure to have a JSON header stating your name, title of your blog post, date, tags, and publication. 
```

Comment:
You're missing the thumbnail attribute. It's only in two files so far but it'll be in all of them soon enough. It is supposed to have the name of an image file inside the `images`  folder